src/main/java/seedu/address/logic/parser/ArgumentMultimap.java

```diff
@@ -59,12 +59,20 @@ public String getPreamble() {
         return getValue(new Prefix("")).orElse("");
     }
 
+    /**
+     * Returns true if none of the prefixes contains empty {@code Optional} values in the
+     * {@code ArgumentMultimap}.
+     */
+    public boolean arePrefixesPresent(Prefix... prefixes) {
+        return Stream.of(prefixes).allMatch(prefix -> this.getValue(prefix).isPresent());
+    }
+
     /**
      * Returns true if none of the prefixes contains empty {@code Optional} values in the given
      * {@code ArgumentMultimap}.
      */
     public static boolean arePrefixesPresent(ArgumentMultimap argumentMultimap, Prefix... prefixes) {
-        return Stream.of(prefixes).allMatch(prefix -> argumentMultimap.getValue(prefix).isPresent());
+        return argumentMultimap.arePrefixesPresent(prefixes);
```

Comment:
Just a nitpick, but is there a reason to overload the method arePrefixesPresent?

Reply:
Nope, but I did it one way and you and Naaman did the static way, but we should standardize in the future.